packages/algob/test/internal/cli/project-creation.ts

```diff
@@ -0,0 +1,60 @@
+import { assert } from "chai";
+import fs from "fs-extra";
+
+import { ERRORS } from "../../../src";
+import { createProject } from "../../../src/internal/cli/project-creation";
+import { expectBuilderErrorAsync } from "../../helpers/errors";
+import { useFixtureProject } from "../../helpers/project";
+
+describe("Init project", () => {
+  useFixtureProject('init-task');
+
+  afterEach(() => {
+    const paths = fs.readdirSync("./");
+    for (const path of paths) {
+      if (path !== "README.md") { fs.removeSync(path); }
+    }
+  });
+
+  it("should init project in a empty folder(javascript)", async () => {
+    const location = "test-project";
+    await createProject(location, false);
```

Comment:
can we use 
```js
 await this.env.run(TASK_INIT, {
   newProjectLocation: location,
   typescript: true/false
 });
 ```   
What do you think?

Reply:
I prefer this one because this is specific to project creation

Reply:
i thought bug was with `INIT`, OK